torchbox/wagtail-torchbox
# Adds fields and template for course landing page

:warning: based on https://github.com/torchbox/wagtail-torchbox/pull/561

### Description of Changes Made

This PR adds the fields needed for the course landing page and configures the subpage listing

### How to Test

1. Check out this branch
2. Add a course landing page
3. add some child course details pages
4. view the parent course landing page from step 2

Note: we don't have a requirement to configure a them for these pages as the designs are only dark, so `dark` has been hard coded as a page property.

Also note: not bothered with yaml as the pattern library is broken for pages

### MR Checklist
